gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -584,6 +569,23 @@ public Builder setAttemptDirectPath(boolean attemptDirectPath) {
       return this;
     }
 
+    /**
+     * Sets a service config for direct path. If direct path is not enabled, the provided service
+     * config will be ignored.
+     *
+     * <p>See <a href=
+     * "https://github.com/grpc/grpc-proto/blob/master/grpc/service_config/service_config.proto">
+     * the service config proto definition</a> for more details.
+     *
+     * <p>This is public only for technical reasons, for advanced usage.
```

Comment:
These two points seem to contradict each other. Rewrite to clarify exactly when and by whom this can be used.

Reply:
Remove this line in favor of the `@InternalApi` annotation.